specification/_global/search/SearchRequest.ts

```diff
@@ -51,69 +51,290 @@ import { RankContainer } from '@_types/Rank'
+     */
     pre_filter_shard_size?: long
+    /**
+     * If `true`, the caching of search results is enabled for requests where `size` is `0`.
+     * Defaults to index level settings.
+     */
     request_cache?: boolean
+    /**
+     * Custom value used to route operations to a specific shard.
+     */
     routing?: Routing
+    /**
+     * Period to retain the search context for scrolling. See Scroll search results.
+     * By default, this value cannot exceed `1d` (24 hours).
+     * You can change this limit using the `search.max_keep_alive` cluster-level setting.
+     * @doc_id scroll-search-results
+     */
     scroll?: Duration
+    /**
+     * How distributed term frequencies are calculated for relevance scoring.
+     */
     search_type?: SearchType
+    /**
+     * Specific `tag` of the request for logging and statistical purposes.
+     */
     stats?: string[]
+    /**
+     * A comma-separated list of stored fields to return as part of a hit.
+     * If no fields are specified, no stored fields are included in the response.
+     * If this field is specified, the `_source` parameter defaults to `false`.
+     * You can pass `_source: true` to return both source fields and stored fields in the search response.
+     */
     stored_fields?: Fields
     /**
      * Specifies which field to use for suggestions.
      */
     suggest_field?: Field
+    /**
+     * Specifies the suggest mode.
+     * This parameter can only be used when the `suggest_field` and `suggest_text` query string parameters are specified.
+     * @server_default missing
+     */
     suggest_mode?: SuggestMode
+    /**
+     * Number of suggestions to return.
+     * This parameter can only be used when the `suggest_field` and `suggest_text` query string parameters are specified.
+     */
     suggest_size?: long
     /**
      * The source text for which the suggestions should be returned.
+     * This parameter can only be used when the `suggest_field` and `suggest_text` query string parameters are specified.
      */
     suggest_text?: string
+    /**
+     * Maximum number of documents to collect for each shard.
+     * If a query reaches this limit, Elasticsearch terminates the query early.
+     * Elasticsearch collects documents before sorting.
+     * Use with caution.
+     * Elasticsearch applies this parameter to each shard handling the request.
+     * When possible, let Elasticsearch perform early termination automatically.
+     * Avoid specifying this parameter for requests that target data streams with backing indices across multiple data tiers.
+     * If set to `0` (default), the query does not terminate early.
+     * @server_default 0
+     */
     terminate_after?: long
+    /**
+     * Specifies the period of time to wait for a response from each shard.
+     * If no response is received before the timeout expires, the request fails and returns an error.
+     * @server_default no timeout
```

Comment:
Is "no timeout" actually a valid value for this property? I'd have assumed it was a numeric value:
```suggestion
     * @server_default 0
```
If we're not sure, I think it's better to omit it.